src/connection/arrayconnection.d.ts

```diff
@@ -78,6 +78,6 @@ export function cursorForObjectInConnection<T>(
  * otherwise it will be the default.
  */
 export function getOffsetWithDefault(
-  cursor?: ConnectionCursor | null,
+  cursor: ConnectionCursor | null | undefined,
```

Comment:
TS complains because `cursor` is optional but the next parameter is not, so I just removed the optional and added undefined to the union. I also synced this on the Flow side.